gen/enum.go

```diff
@@ -134,7 +146,11 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 				<end ->
 				}
 			<end ->
-			return []byte(<$strconv>.FormatInt(int64(<$v>), 10)), nil
+			<if checkErrOnUnknownEnum ->
+				return nil, errors.New(fmt.Sprintf("unknown value for enum %s", <$v>))
+			<else ->	
+				return []byte(strconv.FormatInt(int64(<$v>), 10)), nil
```

Comment:
As before, we either need to use `<$errors>` and `<$strconv>` here, or we can do:

    <import "errors">.New(...)

That will handle ensuring that the package is imported and not import it twice.